_i18n/pl/index.md

```diff
@@ -0,0 +1,190 @@
+                            <div class="col"><h2>Najnowsze wiadomości</h2></div>
+                        </div>
+                        <div class="info-block-row">
+                              {% for post in site.posts limit:2 %}
+                              <div class="row start-xs">
+                                                    <div class="col">
+                                                        <p><a href="{{site.baseurl}}/{{ post.url }}">{{ post.title }}</a></p>
+                                                    </div>
+                                                </div>
+                              {% endfor %}
+                       
+ 
+                            <div class="row end-xs">
+                                <div class="col">
+                                    <p><a href="{{site.baseurl}}/blog/">Więcej</a></p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- end right one-third block-->
+            </div>
+        </section>
+        
+        <!-- END TWO THIRDS & ONE THIRD BLOCKS ROW -->
+        
+        <!-- 3 ONE-THIRD BLOCKS ROW -->
+        
+        <section class="container">
+            <div class="row">
+                <!-- left one-third block-->
+                <div class="left one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>Moneropedia</h2>
+                                <p class="main-resources">Chcesz sprawdzić znaczenie pojęć i konceptów używanych przy Monero? Tutaj znajdziesz alfabetyczny przewodnik do pojęć i ich znaczeń z projektu Monero oraz Kovri.</p>
+                                <p><a href="{{site.baseurl}}/resources/moneropedia/" class="btn-link btn-auto">Przejdź do Moneropedii</a></p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <!-- end left one-third block-->
+                
+                <!-- center one-third block-->
+                <div class="center one-third col-lg-4 col-md-4 col-sm-12 col-xs-12">
+                    <div class="info-block">
+                        <div class="row center-xs">
+                            <div class="col">
+                                <h2>Przewodnik dla użytkowników</h2>
+                                <p class="main-resources">Przewodniki do Monero krok po kroku, podzielone na kategorie. Zawiera wszystko, począwszy od zakładania portfela, do wspomagania sieci, a nawet do tego, jak edytować tę stronę.</p>
+                                <p><a href="{{site.baseurl}}/resources/user-guides/" class="btn-link btn-auto">Przejdź do przewodnika dla użytkowników</a></p>
```

Comment:
This text is too long end it end up outside the button breaking the format, please edit with something shorter